client/cli/src/params/import_params.rs

```diff
@@ -80,6 +80,10 @@ pub struct ImportParams {
 		default_value = "Log"
 	)]
 	pub tracing_receiver: TracingReceiver,
+
+	/// Enable wasm tracing
+	#[structopt(long)]
+	pub wasm_tracing: bool,
```

Comment:
In a case where no `--tracing_targets` are set, the wasm tracing will work - meaning it's activated because you enabled it with `--wasm_tracing`, but the Subscriber will filter out all traces because there are no targets enabled by default.

We could have a default, perhaps allowing all targets with `INFO` or higher level?

FYI proxied traces are a bit different because they're all dispatched from the same callsite (from the viewpoint of tracing) and can't be filtered in the way normal traces are.

Reply:
(The `_` are automatically converted to `-` by structopt for the long name)

Reply:
Is now updated to use the more conventional naming.